### DISCUSSION TITLE
Orca Support Painting tool

### DISCUSSION THREAD

**Discussion by biel112** [OrcaSlicer/OrcaSlicer#7814] [Q&A]
Hi there I am working on a complex part to print, a lot of tree support, but in some place i don't need it.
Just one question if I put Tree  auto as a support, with the Painting Tool I can add some extra tree support, is it possible also remove some support ?

**Comment by LJKraus** [comment]
With  the Support Painting tool (L), use the right mouse button to manually paint the areas where you do _not_ want support.
